doc/pr2_tutorials/kinematics/src/kinematic_model_tutorial.cpp

```diff
@@ -114,7 +108,7 @@ int main(int argc, char **argv)
   // "r_wrist_roll_link" which is the most distal link in the
   // "right_arm" of the robot.
   kinematic_state->setToRandomPositions(joint_model_group);
-  const Eigen::Affine3d &end_effector_state = kinematic_state->getGlobalLinkTransform("r_wrist_roll_link");
+  const Eigen::Affine3d &end_effector_state = kinematic_state->getGlobalLinkTransform("ee_link");
```

Comment:
revert

Reply:
I need to check my branch again. I had a branch for tutorials on a blog post, which I think I forgot to reset to be level with your kinetic-devel before I made some changes. Let me fix this!